src/auro-flight.js

```diff
@@ -102,22 +93,27 @@ class AuroFlight extends LitElement {
         >
         </auro-flight-header>
         <div class="headerContainer">
-          <slot name="departureHeader"></slot>
-          <slot name="arrivalHeader"></slot>
+          ${this.reroutedDepartureStation
+            ? html`<span class="util_displayHiddenVisually">Departure originally</span>` : ''}
```

Comment:
@aesthetiques Did we check the verbiage "Departure originally" and "Arrival originally" with Erika and Deunan? If not lets ping them to see if they are okay with this

Reply:
We will not be including this with our PR, per Erika, we'll allow the owners of that area to determine the verbiage.